### PR TITLE
test: Add e2e workflow tags (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
https://linear.app/n8n/issue/N8N-6073/write-tests-for-workflow-tags

### CODE REVIEW

File: cypress/e2e/17-workflow-tags.cy.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+		wf.getters.tagPills().should('have.length', TEST_TAGS.length);
+		wf.getters.nthTagPill(1).click();
+		wf.actions.addTags('Tag 4');
+		wf.getters.tagPills().should('have.length', TEST_TAGS.length + 1);
+		wf.getters.isWorkflowSaved();
+	});
+
+	it('should create tags via modal', () => {
+		wf.getters.createTagButton().click();
+		wf.getters.tagsDropdown().find('li.manage-tags').first().click();
```
Comment (MiloradFilipovic) [inline cypress/e2e/17-workflow-tags.cy.ts:27]: Can we extract this logic to an action to avoid repeating?

**Comment by MiloradFilipovic** [comment]
Second test (`should create tags via modal`) is failing for me locally
image

**Comment by ivov** [comment]
> Second test (should create tags via modal) is failing for me locally

Unable to reproduce locally but added a small wait to have all tests behave the same.

**Review by MiloradFilipovic — APPROVED** [review]
Approving after testing again.

**Comment by janober** [comment]
Got released with `n8n@0.215.0`

**Pull request merged**